Makefile

```diff
@@ -0,0 +1,8 @@
+
+all: runKompile runJavac
+
+runKompile: semantics/llvm-syntax.k semantics/llvm-syntax-kompiled
+	kompile semantics/llvm-syntax.k --backend java
+
+runJavac: DeleteLLVMComments.java DeleteLLVMComments.class
```

Comment:
Do you really expect to change this program that deletes comments frequently?  I think you will not change it once it works.  In that case, you want to include the .class in the distribution, so that users of your semantics do not need to install and configure javac on their machines.  I would personally get rid of the Makefile completely and use compile to compile the LLVM definition like anything else, and then manually compile the new comment stripper whenever it changes.

Why don't you think a bit through all the minor things above and come up with some solution that makes life easier to your users and to us as well, at the same time.  Maybe talk to other people, too, like Daejun, Andrei, Brandon, Dwight.